ddtrace/propagation/_database_monitoring.py

```diff
@@ -25,23 +26,27 @@
 log = get_logger(__name__)
 
 
-def _default_sql_injector(dbm_comment, sql_statement):
-    # type: (str, str) -> str
+def default_sql_injector(dbm_comment, sql_statement):
+    # type: (str, Union[str, bytes]) -> Union[str, bytes]
     try:
+        if isinstance(sql_statement, bytes):
+            # raises a unicode error when sql statements with non utf-8 encodings
+            # are concatenated. This operation will log a warning.
+            sql_statement = sql_statement.decode("utf-8", errors="strict")
+            return  bytes(dbm_comment + sql_statement, "utf-8")
```

Comment:
I don't think this is correct.

the "source" is `sql_statement` which can be provided as a `str` or a `bytes`, and we want to prepend the `str` `dbm_comment` to `sql_statement`. We shouldn't muck about with encoding/decoding `sql_statement`, we should make `dbm_comment` conform to what `sql_statement` is and then prepend.

wdyt?

Reply:
makes sense to me! https://github.com/DataDog/dd-trace-py/pull/5188/commits/8f156b4cde01c5034f38419ad510313c35373488